docs/DevEnvironment.md

```diff
@@ -108,7 +110,7 @@ The easiest way to install Rouge's development dependencies is using
 [Bundler][]. Bundler is itself a gem but we'll install it differently to how we
 install the other dependencies.
 
-[Bundler]: https://bundler.io/
+[bundler]: https://bundler.io/
```

Comment:
Link matches are [case-insensitive](https://github.github.com/gfm/#matches) and the formatter just picked this up.